src/readers/morphologyASC.cpp

```diff
@@ -217,6 +225,12 @@ class NeurolucidaParser
        Parse the root sexp until finding the first sexp containing numbers
     **/
     Header parse_root_sexp_header() {
+        static const std::map<std::string, Token> NeuriteStringTokenMap{
+            {to_string(Token::AXON), Token::AXON},
```

Comment:
Other than CellBody, do any other tokens have this same behavior?

Reply:
I believe that the old style was only used for CellBody/Soma; I have reduced the scope of the PR to that.

Reply:
Should I update the PR with that in mind?